src/main/connect/connectionManager.ts

```diff
@@ -54,16 +60,20 @@ export class ConnectionManager implements ExtensionComponent, vscode.Disposable
 
     // Minimal version supporting exporting default server configuration.
     private static readonly MINIMAL_JFROG_CLI_VERSION_FOR_DEFAULT_EXPORT: any = semver.coerce('2.6.1');
+    // Minimal version supporting Xsc operations.
+    private static readonly MINIMAL_XSC_VERSION_SUPPORTED: any = semver.coerce('1.7.1');
```

Comment:
the official will be 1.7.7 at least - but not sure it's crucial here, since there won't be 1.7.1 out there

Reply:
changed